velox/functions/prestosql/aggregates/tests/CountAggregationTest.cpp

```diff
@@ -266,5 +270,53 @@ TEST_F(CountAggregationTest, distinctMask) {
           "SELECT c0, count(distinct c2) FILTER (WHERE c1) FROM tmp GROUP BY 1");
 }
 
+TEST_F(CountAggregationTest, nans) {
+  // Verify that NaNs with different binary representations are considered
+  // equal.
+  static const auto kNaN = std::numeric_limits<double>::quiet_NaN();
+  static const auto kSNaN = std::numeric_limits<double>::signaling_NaN();
+  auto data = makeRowVector(
+      {makeFlatVector<int16_t>({1, 2, 1, 2, 1, 1, 2, 2}),
+       // Column to verify with primitive type input
+       makeFlatVector<double>({kNaN, kNaN, kSNaN, kSNaN, 1, 1, 1, 2}),
+       // Column to verify with complex type input
+       makeRowVector(
+           {makeFlatVector<double>({kNaN, kNaN, kSNaN, kSNaN, 1, 1, 1, 2}),
+            makeFlatVector<int32_t>(std::vector<int32_t>(8, 1))})});
```

Comment:
nit: its very easy to confuse (8, 1) with {8, 1} , if possible can you just make this {1 , 1, 1, ...} so its consistent

Reply:
will do!